CHANGELOG.md

```diff
@@ -1,5 +1,219 @@
 # Changelog
 
+## 5.0.0.rc1
+
+Teleport 5.0 is a major release with new features, functionality, and
+bug fixes. Users can review [5.0 closed issues](https://github.com/gravitational/teleport/milestone/39?closed=1)
+on Github for details of all items.
+
+#### New Features
+Teleport 5.0 is a major version release and introduces two distinct features: Teleport Application Access and the Unified Access Plane with significant Kubernetes enhancements like multi-cluster support, backwards compatibility, and more.
+and the new addition of Application Support.
+
+##### Teleport Application Access
+Teleport provides secure access to web applications. This new feature was built with the express intention of securing those internal apps which might once lived on a VPN or had a simple oauth mechanism with little to no audit trail. We've tested everything from various dashboards to single page javascript apps.
+
+Adding an application follows the same UX as adding SSH servers or Kubernetes clusters, starting with creating a static or dynamic invite token.
+
+```bash
+$ tctl tokens add --type=app
+```
+
+A Teleport node can be used to route an application from the unified access plane.
+The application access service can be deployed alongside applications, enabling teams
+to setup access on a loopback address so only application access can provide ingress to
+the app.
+
+```yaml
```

Comment:
This goes into  a lot of detail really quick, we have a simple CLI flow, let's mention it first.

Reply:
Added in this sentance

> > 
> Adding an application is as simple as starting Teleport with a few new flags.
> ```sh
> $ teleport start --roles=app --token=xyz --auth-server=proxy.example.com:3080 \
>     --app-name="example-app" \
>     --app-uri="http://localhost:8080"
> ```
> This start script will create an app server that proxies the application "example-app"
> running at http://localhost:8080.
> 
> Applications can can configured using the new `app_service`.
>